src/main/java/com/nextstep/lotto/view/InputView.java

```diff
@@ -0,0 +1,20 @@
+package com.nextstep.lotto.view;
+
+import java.util.Scanner;
+
+public class InputView {
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    public static int inputPrice() {
+        System.out.println("구입금액을 입력해 주세요.");
+        String line = SCANNER.nextLine();
+        return Integer.parseInt(line);
```

Comment:
```suggestion
        return SCANNER.nextInt();
```
바로 int 타입으로 반환해도 좋을 것 같네요